site/en/blog/new-in-chrome-87/index.md

```diff
@@ -0,0 +1,243 @@
+---
+title: New in Chrome 87
+description: >
+  Chrome 87 is rolling out now! You can now control pan, tilt, and zoom on
+  webcams that support it, range requests and service workers don’t require
+  as many workarounds, the font access API starts it’s origin trial, and
+  plenty more. Let’s dive in and see what’s new for developers in Chrome 87!
+layout: 'layouts/blog-post.njk'
+date: 2020-11-17
+updated: 2020-11-23
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/xFjaDDkGd2mBReqdPxdy.png'
+alt: >
+  New in Chrome hero logo
+---
+
+Chrome 87 is starting to roll out to stable now.
+
+{% youtube id='NCKMMzVn1c8' %}
```

Comment:
Is there any way to get this to appear half-width on wide screen (similar to https://developers.google.com/web/updates/2020/11/nic87)